src/common/stats/StatsManager.inl

```diff
@@ -19,9 +19,9 @@ StatsManager::VT StatsManager::readValue(StatsHolder& stats,
         case StatsMethod::COUNT:
             return stats.count(level);
         case StatsMethod::AVG:
-            return stats.avg(level);
+            return stats.template avg<VT>(level);
```

Comment:
Why the changes?

Reply:
The default type argument here is `double` while `VT` is of type `int64_t`, thus introducing two meaningless type conversions.